addons/pos_restaurant/static/src/app/screens/floor_screen/floor_screen.js

```diff
@@ -217,12 +219,15 @@ export class FloorScreen extends Component {
                     return;
                 }
                 const oToTrans = this.pos.getActiveOrdersOnTable(table)[0];
-                if (oToTrans) {
-                    this.pos.transferOrder(oToTrans.uuid, this.state.potentialLink.parent);
-                }
                 this.pos.data.write("restaurant.table", [table.id], {
                     parent_id: this.state.potentialLink.parent.id,
                 });
+                this.pos.data.write("restaurant.table", [table.parent_id.id], {
+                    child_ids: [[4, this.state.potentialLink.child.id]],
+                });
+                if (oToTrans) {
+                    this.pos.transferOrder(oToTrans.uuid, this.state.potentialLink.parent, true);
+                }
```

Comment:
why did you move this here?

Reply:
actually, we were trying to update the prep card after merging of table so at that time we needed `parent_id` data before `transferOrder` 
but now we don't update prep card. I will keep it the same as it was earlier.